PlusMinus.cs

```diff
@@ -0,0 +1,51 @@
+using System.CodeDom.Compiler;
+using System.Collections.Generic;
+using System.Collections;
+using System.ComponentModel;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Serialization;
+using System.Text.RegularExpressions;
+using System.Text;
+using System;
+using static System.Console;
+
+class Result
+{
+    public static void plusMinus(List<int> arr)
+    {
+        var  positive=0; 
+        var  negative=0;
+        var  zero=0;
```

Comment:
initialize with double. This will prevent casting in line numbers 33, 34, 35 @samradhshrma

Reply:
@narayanagunturi  I have resolved this problem.